include/mbgl/mtl/renderer_backend.hpp

```diff
@@ -0,0 +1,67 @@
+#pragma once
+
+#include <mbgl/gfx/renderer_backend.hpp>
+#include <mbgl/mtl/mtl_fwd.hpp>
+#include <mbgl/util/image.hpp>
+#include <mbgl/util/size.hpp>
+#include <mbgl/util/util.hpp>
+
+#include <Foundation/NSSharedPtr.hpp>
+#include <Metal/MTLDevice.hpp>
+#include <Metal/MTLCommandQueue.hpp>
+
+namespace mbgl {
+
+class ProgramParameters;
+
+namespace mtl {
+
+using ProcAddress = void (*)();
+using FramebufferID = uint32_t;
+
+class RendererBackend : public gfx::RendererBackend {
+public:
+    RendererBackend(gfx::ContextMode);
+    ~RendererBackend() override;
+
+    /// Called prior to rendering to update the internally assumed OpenGL state.
```

Comment:
Should be 'Metal state'
```suggestion
    /// Called prior to rendering to update the internally assumed Metal state.
```